Changed the explanation of the NTBias Diagram

wbarobinson opened the pull request:
I believe (and please correct me if I am wrong) the explanation that the diagram is describing a 34% nay requirement for rejection is backwards. With a negative turnout bias, the whole point is that with few people it takes a a huge percentage to reject (or vice-versa, a small amount of ayes to accept). So when there is only 25% percent turnout in a negative turnout bias situation, the ayes only need 34% of the vote to carry the council's proposal through.

lsaether commented on docs/polkadot/learn/staking.md:
```diff
@@ -70,6 +72,31 @@ Rewards can be used the same account (controller) to keep accumulating the rewar
+
+**Blue line**: return rate based on the staking participation
+
+**Green line**: how many percentages of DOTs being staked to the network and the corresponding inflation rate.
```
```suggestion
**Green line**:  inflation rate
```

Can you define the x-axis and y-axis separately too?

ansonla3 replied:
sure

lsaether commented on docs/polkadot/learn/staking.md:
```diff
@@ -70,6 +72,31 @@ Rewards can be used the same account (controller) to keep accumulating the rewar
+
+**Green line**: how many percentages of DOTs being staked to the network and the corresponding inflation rate.
+
+The above chart shows the inflation model of the network. Depends on the staking participation, the inflation rate will be dynamically changed to incentivize / disincentivize token holders to participate in the staking, for instance, inflation would be changed to 10% if there is 50% of DOTs being staked to the network.
```
```suggestion
The above chart shows the inflation model of the network. Depending on the staking participation, the inflation rate will be dynamically changed to incentivize / disincentivize token holders to participate in staking. For instance, inflation would be 10% if there is 50% of DOTs being staked to the network.
```

lsaether commented on docs/polkadot/learn/staking.md:
```diff
@@ -70,6 +72,31 @@ Rewards can be used the same account (controller) to keep accumulating the rewar
+
+The above chart shows the inflation model of the network. Depends on the staking participation, the inflation rate will be dynamically changed to incentivize / disincentivize token holders to participate in the staking, for instance, inflation would be changed to 10% if there is 50% of DOTs being staked to the network.
+
+On the other hand, come up with the ideal staking rate is not an easy task since the network itself requires to have enough amount of DOTs being staked to provide the security guarantee and to avoid illiquidity for the market. 
```
```suggestion
Determining the ideal staking rate is not an easy task as the network requires enough DOTs to be staked to provide the security guarantees we want and to avoid illiquidity on the market. 
```

wbarobinson committed:
Changed the explanation of the NTBias Diagram

I believe (and please correct me if I am wrong) the explanation that the diagram is describing a 34% nay requirement for rejection is backwards. With a negative turnout bias, the whole point is that with few people it takes a a huge percentage to reject (or vice-versa, a small amount of ayes to accept). So when there is only 25% percent turnout in a negative turnout bias situation, the ayes only need 34% of the vote to carry the council's proposal through.

lsaether commented:
#152
